test/unit/sanitizer/sanitizeAll.js

```diff
@@ -140,11 +147,97 @@ module.exports.tests.all = function(test, common) {
       warnings: []
     };
 
-    sanitizeAll(req, sanitizers, function(){
-      t.deepEquals(req, expected_req);
-      t.end();
-    });
+    sanitizeAll.runAllChecks(req, sanitizers);
+    t.deepEquals(req, expected_req);
+    t.end();
+  });
+
+  test('unexpected parameters should throw warning', function(t) {
```

Comment:
Using the word throw is misleading because it usually implies am exception is getting thrown and needs to be caught. Could lead to misunderstanding of the code